internal/action/show.go

```diff
@@ -174,7 +174,7 @@ func (s *Action) showGetContent(ctx context.Context, sec gopass.Secret) (string,
 		debug.Log("got(found: %t) key %s: %s", found, key, val)
 		return val, val, nil
 	} else if HasKey(ctx) {
-		out.Warning(ctx, "parsing is disabled but a key was provided.")
+		out.Warning(ctx, "⚠ Parsing is disabled but a key was provided.")
```

Comment:
Maybe add "⚠" to the prefix of `out.Warning` instead?

Reply:
You've actually already done so, so this ⚠ can be removed here.

Reply:
Done.